src/web/components/label/compliancestate.jsx

```diff
@@ -0,0 +1,91 @@
+/* SPDX-FileCopyrightText: 2024 Greenbone AG
+ *
+ * SPDX-License-Identifier: AGPL-3.0-or-later
+ */
+
+import styled from 'styled-components';
+
+import useTranslation from 'web/hooks/useTranslation';
+import {styledExcludeProps} from 'web/utils/styledConfig';
+import Theme from 'web/utils/theme';
+
+const Label = styledExcludeProps(styled.div, [
+  'backgroundColor',
+  'borderColor',
+])`
+  text-align: center;
+  font-weight: normal;
+  font-style: normal;
+  color: white;
+  padding: 1px;
+  display: inline-block;
+  width: 70px;
+  height: 1.5em;
+  font-size: 0.8em;
+  background-color: ${props => props.backgroundColor};
+  border-color: ${props => props.borderColor};
+`;
+
+
+const YesLabel = props => {
+  const [_] = useTranslation();
+  return (
+  <Label
+    {...props}
+    backgroundColor={Theme.complianceYes}
+    borderColor={Theme.complianceYes}
+  >
+    {_('Yes')}
+  </Label>
+  )
+};
+
+const NoLabel = props => {
+  const [_] = useTranslation();
+  return (
+  <Label
+    {...props}
+    backgroundColor={Theme.complianceNo}
+    borderColor={Theme.complianceNo}
+  >
+    {_('No')}
+  </Label>
+  )
+};
```

Comment:
I think we can create one generic component and then export each case with the props passed.

```js
const YesLabel = props => <ComplianceLabel {...props} text='Yes' color='complianceYes' />;
```

Reply:
Good idea. Done